packages/api/src/routes/internal/index.ts

```diff
@@ -47,13 +47,15 @@ import commonwellRoutes from "./hie/commonwell";
 import jwtToken from "./jwt-token";
 import docsRoutes from "./medical/docs";
 import facilityRoutes from "./medical/facility";
+import ffsRoutes from "./medical/feature-flags";
 import feedbackRoutes from "./medical/feedback";
 import mpiRoutes from "./medical/mpi";
 import organizationRoutes from "./medical/organization";
 import patientRoutes from "./medical/patient";
 
 const router = Router();
 
+router.use("/feature-flags", ffsRoutes);
```

Comment:
should be a singular 'feature-flag' as it corresponds to the entity and operations on the entity.

nit, but similarly ffRoutes.

Reply:
I understand what you mean, but "feature flags" is a diff beast as we have a single payload with all feature flags. It's not like we're managing a single FF on this endpoint.